### DISCUSSION TITLE
Button bar does is not displayed on the Playback screen.

### DISCUSSION THREAD

**Discussion by Sagnasty** [rern/rAudio#73] [Fixed Issues]
Using rAudio 1 • 202101021, the Buttons bar is not displayed on the Playback display with iPhone SE 2 (portrait and landscape mode).

I have verified that the display Buttons selection is active in the "show selected items" list.

The Buttons bar behavior is correctly displayed on windows machine with Firefox browser using the same RPi system.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Comment by rern** [comment]
Limited display area needs limited items to display.
If you like, everything can be done without top/bottom bars, time/volume knobs and buttons. Tap coverart top to see guide.

**Reply by Sagnasty** [reply]
Yes the controls guide will work, but please consider the following:

For me, the playback screen was more intuitive when the items were "stacked" in a single column while in portrait mode with small screen with limited hidden controls (i.e. what we had in RuneAudio+R).  I could simply scroll up / down to view the item of interest if the display width was limited.   I could also select which controls should be displayed or not displayed and could verify that the change took effect.

When a user selects the button bar (or any other control) to be visible, then it should be visible for all circumstances (otherwise, they will think it is a bug). 

Perhaps what is needed is a selection for which Playback view the user would like (Playback view with more hidden features (volume / time elapsed sliders, controls guide, etc.) or instead more towards RuneAudio+R where most Playback controls were not hidden and single column stackink with up/down scrolling.

Either approach is perfectly valid, but a mix of both can quickly become confusing to use and not user friendly.

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+
